config/webpack.vendor.js

```diff
@@ -1,73 +1 @@
-module.exports = [
-  /**
-   * React related
-   */
-  'babel-polyfill',
-  'react',
-  'react-dom',
-  'react-debounce-input',
-  'react-diff-view',
-  'react-ellipsis-with-tooltip',
-  'react-numeric-input',
-  'react-onclickoutside',
-  'react-password-strength',
-  'patternfly-react',
-  'react-redux',
-  'redux',
-  'redux-form',
-  'redux-form-validators',
-  'redux-logger',
-  'redux-thunk',
-  'prop-types',
-  'seamless-immutable',
-  'isomorphic-fetch',
-  'react-intl',
-  'intl',
-  'react-router-dom',
-  'history',
-
-  /*
-   * jquery related
-   */
-  'jquery',
-  'jquery.cookie',
-  'select2',
-  'multiselect',
-  'jquery-ujs',
-  'datatables.net-bs',
-  // jquery-flot related
-  'jquery-flot/excanvas',
-  'jquery-flot',
-  'jquery-flot/jquery.flot.pie',
-  'jquery-flot/jquery.flot.selection',
-  'jquery-flot/jquery.flot.stack',
-  'jquery-flot/jquery.flot.time',
-
-  /**
-   * Brace related
-   */
-  'brace',
-  'brace/mode/ruby',
-  'brace/mode/html_ruby',
-  'brace/mode/json',
-  'brace/mode/sh',
-  'brace/mode/xml',
-  'brace/mode/yaml',
-  'brace/mode/diff',
-  'brace/theme/twilight',
-  'brace/theme/clouds',
-  'brace/keybinding/vim',
-  'brace/keybinding/emacs',
-  'brace/ext/searchbox',
-
-  /**
-   * Other packages
-   */
-  'axios',
-  'diff',
-  'ipaddr.js',
-  'jstz',
-  'urijs',
-  'uuid',
-  '@novnc/novnc/core/rfb.js',
-];
+module.exports = ['react-intl', 'intl'];
```

Comment:
why should we keep foreman vendor at all at this stage?

Reply:
Because I want the first version of the `vendor` to be a small increment of what we have in foreman atm.
Because the `intl` packages cause webpack to do dynamic imports and split into multiple files I would prefer to avoid it in the first version.